lib/src/repository.rs

```diff
@@ -550,6 +637,17 @@ impl OmicronRepoEditor {
     }
 }
 
+/// Whether to include the installinator document.
+///
+/// TODO-cleanup: This is a transitional option for v15 -> v16, meant to be used
+/// for testing. After v16 we can assume that all valid TUF repositories have
+/// installinator documents.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum IncludeInstallinatorDocument {
+    Yes,
+    No,
+}
```

Comment:
This really seems like it should be a bool.

Reply:
I personally tend to avoid using bools in public APIs because it can be hard to tell what a particular bool means when looking at code. I don't feel strongly about this though.